### PR TITLE
Adding for-reviewers.hackclub.com

### PR DESCRIPTION
**Pull request by souptik-samanta**
#Adding for-reviewers.hackclub.com

##I and few others(given credits at the bottom of page) made a website to show a small gesture of love for Reviewer's hard work

It will be used for hosting my website that is for the reviewers here is the git pages link https://souptik-samanta.github.io/final-for-arcade/index.html
Screenshot (226)

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2954,3 +2954,7 @@ shipment-viewer:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+for-reviewers:
+   - ttl: 600
+    type: CNAME
+    value: souptik-samanta.github.io/final-for-arcade/index.html.
```
Comment (Muirrum) [inline hackclub.com.yaml:2960]: DNS records can't have paths (`/final-for-arcade/index.html`)

```suggestion
    value: souptik-samanta.github.io.
```

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 1 above)
Comment (souptik-samanta) [reply hackclub.com.yaml:2960]: Hey but if i change to souptik-samanta.github.io then this thing is showing
what should i do?

image

Comment (Muirrum) [reply hackclub.com.yaml:2964]: You need to configure github pages https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages#using-a-subdomain-for-your-github-pages-site

File: hackclub.com.yaml [hunk 2]
```diff
@@ -2958,3 +2958,7 @@ shipment-viewer:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+for-reviewers:
+   - ttl: 600
+    type: CNAME
+    value: souptik-samanta.github.io.
```
Comment (Muirrum) [inline hackclub.com.yaml:2964]:
```suggestion
   - ttl: 600
     type: CNAME
     value: souptik-samanta.github.io.
```

just need to align the indentation

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 2 above)
Comment (souptik-samanta) [reply hackclub.com.yaml:2964]: Done

**Comment by souptik-samanta** [comment]
Yes did it

File: hackclub.com.yaml (hunk 2 above)
Comment (souptik-samanta) [reply hackclub.com.yaml]: hey just changed that

**Review by Muirrum — APPROVED** [review]
lgtm

**Comment by souptik-samanta** [comment]
thnx

**Pull request merged**